src/lib/storage/storage.ts

```diff
@@ -0,0 +1,29 @@
+import { serialize, deserialize } from "bson"
+import fs from "fs"
+import os from "os"
+import path from "path"
+
+import { Storage } from "./types.js"
+
+const dataDir = path.resolve(os.tmpdir(), "rune")
```

Comment:
I don't know enough about temp files, but might want to put them in a more permanent place to avoid devs having to reauthenticate (when we add email verification) if they only upload a game ever so often: https://serverfault.com/a/906501

Reply:
Good point, I'll think about it

Reply:
@bfelbo changed it to using OS-specific hode directory https://github.com/rune/rune-games-cli/pull/10/commits/fe1d9900677687361325b9f9376266b988142519